types/booking.d.ts

```diff
@@ -53,3 +44,23 @@ export interface Booking {
   /** flight offer for booking */
   flightOffer: FlightOffer;
 }
+
+/**
+ * Interface of a booking returned from the database
+ */
+export interface DatabaseBooking extends Booking {
```

Comment:
`DatabaseBooking` sounds a bit ... dry 🥴 What do you think about naming this class `Booking` and the other is more a `BookingDraft` ? @antonplagemann